bookkeeper-server/src/main/java/org/apache/bookkeeper/client/LedgerEntry.java

```diff
@@ -1,114 +0,0 @@
-/*
- *
- * Licensed to the Apache Software Foundation (ASF) under one
- * or more contributor license agreements.  See the NOTICE file
- * distributed with this work for additional information
- * regarding copyright ownership.  The ASF licenses this file
- * to you under the Apache License, Version 2.0 (the
- * "License"); you may not use this file except in compliance
- * with the License.  You may obtain a copy of the License at
- *
- *   http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the License is distributed on an
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- * KIND, either express or implied.  See the License for the
- * specific language governing permissions and limitations
- * under the License.
- *
- */
-package org.apache.bookkeeper.client;
-
-import com.google.common.base.Preconditions;
-import io.netty.buffer.ByteBuf;
-import io.netty.buffer.ByteBufInputStream;
-
-import java.io.InputStream;
-import org.apache.bookkeeper.conf.ClientConfiguration;
-
-/**
- * Ledger entry. Its a simple tuple containing the ledger id, the entry-id, and
- * the entry content.
- *
- */
-public class LedgerEntry
```

Comment:
I think you still need to keep this class for BC issue.

you can extend it from LedgerEntryImpl and override the getEntry() and keep getEntryInputStream(). because getEntry and getEntryInputStream should stay same semantics as legacy semantic in this class.

```
public class LedgerEntry extends LedgerEntryImpl {

     

}
```